deploy/complete/helm-chart/provision/values.yaml

```diff
@@ -9,7 +9,7 @@ global:
     # Set object storage namespace, usualy the tenancy name or ocid.
     objectstoragenamespace: ocitenancyname
     # Service broker name (used in the service URL)
-    name: mushop-osb
+    name: oci-broker
```

Comment:
Change required otherwise the instructions fail.

Reply:
ok. missed on the previous PR.